client/packages/invoices/src/InboundShipment/DetailView/AddButton.tsx

```diff
@@ -67,7 +70,9 @@ export const AddButton = ({
         onAddItem(true);
         break;
       case 'add-from-master-list':
-        masterListModalController.toggleOn();
+        invoice?.status === InvoiceNodeStatus.New
+          ? masterListModalController.toggleOn()
+          : error(t('error.cannot-add-from-masterlist-if-not-new'))();
```

Comment:
Could you change this to use info() rather than error() - we're just letting the user know why the action is unavailable, as opposed to something has gone wrong 🙏